### ISSUE TITLE
Cannot read property 'toLowerCase' of undefined at NgImageSliderComponent.handleKeyboardEvent (ng-image-slider.js:181)

### ISSUE THREAD

**Issue by illyakoryagin12** [sanjayV/ng-image-slider#9]
I have material controls(mat-form-field, mat-input). When I select an item from the autocomplete list and click on the item I get this error.

**Comment by hpardess** [comment]
I faced same error
```
ImageSliderComponent.html:4 ERROR TypeError: Cannot read property 'toLowerCase' of undefined
    at NgImageSliderComponent.push../node_modules/ng-image-slider/fesm5/ng-image-slider.js.NgImageSliderComponent.handleKeyboardEvent (ng-image-slider.js:181)
```
I investigated the error and found following piece of code library causes this error :
ng-image-slider/fesm5/ng-image-slider.js
<img width="732" alt="Screen Shot 1398-03-01 at 12 26 53 AM" src="https://user-images.githubusercontent.com/6196491/58127010-9d88c380-7c29-11e9-9ad7-214115c40b73.png">

the event variable is undefined. It is required to add a check like,
```
if(event == undefined) {
     return;
}
```
@sanjayV : Please fix and release as soon as possible to update it in our project.

**Comment by sanjayV** [comment]
@illyakoryagin12 @hpardess 
due to busy schedule I was not able to look on issues, 
but yes, I will provide you updated version soon

**Comment by sanjayV** [comment]
@illyakoryagin12 @hpardess 
FYI: I have released new version (1.3.1) with fixes

**Comment by illyakoryagin12** [comment]
problem is solved in the new version (1.3.1). Thanks, @sanjayV
